clippy_lints/src/utils/conf.rs

```diff
@@ -172,6 +172,8 @@ define_Conf! {
     (enum_variant_name_threshold, "enum_variant_name_threshold", 3 => u64),
     /// Lint: LARGE_ENUM_VARIANT. The maximum size of a emum's variant to avoid box suggestion
     (enum_variant_size_threshold, "enum_variant_size_threshold", 200 => u64),
+    /// Lint: VERBOSE_BIT_MASK. The maximum size of a bit mask, that won't be checked on verbosity
```

Comment:
That comment should be reworded to mention what the lint does instead of "verbosity". Without already knowing the lint, that comment does not shed any light on what the setting will change

Reply:
Replaced on "The maximum size of a bit mask, when it wouldn't be suggested to call 'trailing_zeros' instead" . What do you think?